core/src/main/scala/kafka/log/UnifiedLog.scala

```diff
@@ -1945,26 +1947,17 @@ object UnifiedLog extends Logging {
       logOffsetsListener)
   }
 
-  def logFile(dir: File, offset: Long, suffix: String = ""): File = LogFileUtils.logFile(dir, offset, suffix)
```

Comment:
This and other deleted methods were simply passing through to `LogFileUtils` and hence did not add enough value to retain.